File: backend/components/metiers/administration_publique/templates/helpers/validators/templates_validator.test.js
// templates_validator.test.js – Tests unitaires JS pour validateurs templates Threed
const validator = require('./templates_validator');
describe('Validator templates Threed (JS)', () => {
  test('isValidTemplateFile fonctionne', () => {
    expect(typeof validator.isValidTemplateFile).toBe('function');
  });
  test('validateTemplateContent fonctionne', () => {
    expect(typeof validator.validateTemplateContent).toBe('function');
  });
});

File: backend/components/metiers/administration_publique/templates/helpers/validators/templates_validator.test.py
# templates_validator.test.py – Tests unitaires Python pour validateurs templates Threed
import unittest
from backend.components.metiers.threed.templates.helpers.templates_validator import is_valid_template_file, validate_template_content

class TestTemplatesValidator(unittest.TestCase):
    def test_is_valid_template_file(self):
        self.assertTrue(callable(is_valid_template_file))
    def test_validate_template_content(self):
        self.assertTrue(callable(validate_template_content))

if __name__ == '__main__':
    unittest.main()
